docs/v0.5/getting-started/docker-for-windows.md

```diff
@@ -365,8 +367,72 @@ riff core deployer delete k8s-square
 riff function delete square
 ```
 
-## Uninstalling and reinstalling
+## Uninstalling
+
+Use the following commands to uninstall riff:
+
+### Remove any riff resources
+
+```powershell
+kubectl delete riff --all-namespaces --all
+```
+
+### remove riff Streaming Runtime
+
+```powershell
+kapp delete -n apps -a riff-streaming-runtime
+```
+
+```powershell
+kapp delete -n apps -a keda
+```
+
+### remove riff Core Runtime (if installed)
+
+```powershell
+kapp delete -n apps -a riff-core-runtime
+```
+
+### remove riff Knative Runtime (if installed)
+
+```powershell
+kubectl delete knative --all-namespaces --all
+```
+
+```powershell
+kapp delete -n apps -a riff-knative-runtime
+```
+
+```powershell
+kapp delete -n apps -a knative
+```
+
+```powershell
+kapp delete -n apps -a istio
+```
+
+```powershell
+kubectl get customresourcedefinitions.apiextensions.k8s.io -oname | grep istio.io | xargs -L1 kubectl delete
+```
+
+### remove riff Build
+
+```powershell
+kapp delete -n apps -a riff-build
+```
+
+```powershell
+kapp delete -n apps -a riff-builders
+```
+
+```powershell
+kapp delete -n apps -a kpack
+```
+
+```powershell
+kapp delete -n apps -a cert-manager
+```
 
-If you need to upgrade or reinstall riff, we recommend resetting the Kubernetes cluster first. To do this, click `Reset Kubernetes Cluster...` in the Reset tab in Docker Settings.
+Alternatively you an reset the Kubernetes cluster (this will remove all state including riff).
```

Comment:
Maybe we should put this under a different heading? Right now it looks like it is grouped with `remove riff Build`

Proposal:
```
### reset cluster
As an alternative to uninstalling with `kapp`, you can reset the Kubernetes cluster ....
````

Reply:
either way: "you an" -> "you can"